src/Symfony/Component/Mailer/Bridge/Sendgrid/Tests/Transport/SendgridApiTransportTest.php

```diff
@@ -39,6 +39,10 @@ public static function getTransportData()
                 new SendgridApiTransport('KEY'),
                 'sendgrid+api://api.sendgrid.com',
             ],
+            [
+                new SendgridApiTransport('KEY', region: 'eu'),
```

Comment:
the test should not use a named argument IMO, given that we don't guarantee BC on named arguments but only on positional ones (and it is better if the test uses our public API as it would detect cases where we break it)